delfin/drivers/dell_emc/unity/unity.py

```diff
@@ -410,15 +410,13 @@ def list_qtrees(self, context):
                 qts_entries = qts.get('entries')
                 for qtree in qts_entries:
                     content = qtree.get('content', {})
```

Comment:
If `content` could not be got, it seems that there is no need to continue to parse the empty dict and append a empty `qt` to `qt_list`

Reply:
Done, but in fact,content always will not be none